helm/helm.go

```diff
@@ -81,6 +81,10 @@ func main() {
 					Name:  "force, aye-aye",
 					Usage: "Perform install even if dependencies are unsatisfied.",
 				},
+				cli.BoolFlag{
+					Name:  "dry-run",
+					Usage: "Run but do not install into Kubernetes.",
```

Comment:
@sgoings I was just thinking of what many other programs do during _dry run_ operations. It feels like we'd want to step as far through the process as possible without actually making any changes to Kubernetes.